classes/Cart.php

```diff
@@ -500,6 +500,9 @@ public function getCartRules($filter = CartRule::FILTER_ACTION_ALL, $autoAdd = t
                     AND crl.id_lang = ' . (int) $this->id_lang . '
                 )
                 WHERE `id_cart` = ' . (int) $this->id . '
+                AND NOW() BETWEEN cr.date_from AND cr.date_to
+				AND cr.`active` = 1
+				AND cr.`quantity` > 0
```

Comment:
```suggestion
                AND NOW() BETWEEN cr.date_from AND cr.date_to
                AND cr.`active` = 1
                AND cr.`quantity` > 0
```

But I'm afraid you break integration tests, you change is breaking something :)

Reply:
@Progi1984  Ok, I let you have a look on a manually test ;)